app/controllers/editions_controller.rb

```diff
@@ -187,7 +187,7 @@ def review
   def destroy
     if resource.can_destroy?
       destroy! do
-        flash[:success] = "#{description(resource)} destroyed"
+        flash[:success] = "Edition deleted"
```

Comment:
Other flash messages in this file tend to show the format of the edition – i.e. the `#{description(resource)}` bit that you removed.

For example, when a new Edition is created:

https://github.com/alphagov/publisher/blob/572213a6f5a595c00546ce185b4b6367c938159b/app/controllers/editions_controller.rb#L54

So when creating a new help page edition, the flash message would read:
**Help page successfully created**

But this change means the delete action will always just read:
**Edition deleted**

regardless of the type of edition being deleted.

I'm just flagging it as I believe it introduces an inconsistency into the flash messages shown to users when working on editions.

Reply:
Thanks @ollietreend. We think this change is fine because, in your example of creating a new artefact, you are creating a new thing (for example, a guide) in comparison to deleting an edition (of a thing).